README.md

```diff
@@ -20,14 +20,16 @@ in Gitea. This is done by utilizing the extension points for [automatic git prov
 instance:
   ```bash
   #!/bin/bash
+  VERSION=0.1.0
   NAMESPACE=default
```

Comment:
Can be removed since it is not used in this code block.

Reply:
Version is used in the URL for the helm chart artifact: ```https://github.com/keptn-sandbox/keptn-gitea-provisioner-service/releases/download/${VERSION}/keptn-gitea-provisioner-service-${VERSION}.tgz```

Reply:
Sorry, I did not mean the version but the `NAMESPACE`, but maybe we should use the `NAMESPACE` for the `GITEA_ENDPOINT`. If not, you can remove it.